doc/changelog/04-tactics/18887-rm-deprecated-clear-modifier.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  the clear modifier which was deprecated since 8.17
```

Comment:
```suggestion
  the clear modifier which has been deprecated since 8.17
```

Reply:
Despite having a similar behaviour, this is not the clear tactic, so I'm not sure we want to do this. The previous changelog entry did not have a special rendering for this word (see #16407).